src/common/threadable.cc

```diff
@@ -11,7 +11,7 @@ Threadable::Threadable()
 {}
 
 Threadable::~Threadable()
-{}
+= default;
```

Comment:
I tend to do the previous thing we had here too - what is the difference?

Reply:
Nothing -- it's just a style thing. Now that you point it out, I see that it's a pretty dubious improvement at best :thinking:.

Let's just disable this warning.